fabric-item-api-v1/src/main/java/net/fabricmc/fabric/impl/item/FabricItemInternals.java

```diff
@@ -43,8 +45,8 @@ public static void onBuild(Item.Settings settings, Item item) {
 	}
 
 	public static final class ExtraData {
-		private /* @Nullable */ EquipmentSlotProvider equipmentSlotProvider;
-		private /* @Nullable */ CustomDamageHandler customDamageHandler;
+		private EquipmentSlotProvider equipmentSlotProvider;
```

Comment:
These too are nullable, I believe this was coded back when nullable annotations were not used?